packages/curve-ui-kit/.storybook/preview.tsx

```diff
@@ -41,10 +52,11 @@ export const decorators: Decorator[] = [
           src: url('fonts/Minecraft-Bold.otf') format('opentype');
         }
         `}
-      </style>
-      <Story />
-    </>
-  ),
+        </style>
+        <RouterProvider router={router} />
```

Comment:
Shouldn't this be removed from the ManageSoftLiquidation story?
https://github.com/curvefi/curve-frontend/blob/4315549d5d9ed73943609ba24b6238f7f21abd3d/apps/main/src/llamalend/features/manage-soft-liquidation/ManageSoftLiquidation.stories.tsx#L60